CommonClient.py

```diff
@@ -32,6 +34,7 @@
 # without terminal, we have to use gui mode
 gui_enabled = not sys.stdout or "--nogui" not in sys.argv
 
+ssl_context = ssl.create_default_context(ssl.Purpose.SERVER_AUTH, cafile=certifi.where())
```

Comment:
I am a bit concerned about this firing on import, it's probably fine, but making a note here.

Reply:
You are right. Does the new code look better?

Reply:
It does, thank you.